src/components/Calculator.js

```diff
@@ -1,71 +1,57 @@
-import React, { Component } from 'react';
+/* eslint-disable react/prefer-stateless-function */
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 import Button from './Button';
 
-class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      obj: {
-        total: null,
-        next: null,
-        operation: null,
-      },
-    };
-    this.handleClick = this.handleClick.bind(this);
-  }
+export default function Calculator() {
+  const [calcObj, setCalcObj] = useState({
```

Comment:
- [ ] Please, consider using arrow function syntax 👍 as it is shown in an article in project requirements - I know you can do it easily 🥇 

```
const Calculator = () => {
....
}

export default Calculator;
```

Reply:
Hello @elmar8287  thanks so much for the professional code review, 
I made what was required,  using arrow function syntax.
You can now check